sdcm/nemesis.py

```diff
@@ -873,38 +850,17 @@ def _add_drop_column_run_in_cycle(self):
 
     def disrupt_add_drop_column(self):
         """
-        It searches for table that allow add/drop columns (non compact storage table) and create it if no such table.
-        If table is not self-created, add and/or drop columns from that table and quit.
-        If table is self-created, run cassandra-stress for 3 minutes on it and,
-            in the same time run cycle of adding and/or dropping columns.
+        It searches for a table that allow add/drop columns (non compact storage table)
+        If there is no such table it draw an error and quit
         It keeps tracking what columns where added and never drops column that were added by someone else.
         """
         self.log.debug("AddDropColumnMonkey: Started")
         self._add_drop_column_target_table = self._add_drop_column_get_target_table(
             self._add_drop_column_target_table)
-        if self._add_drop_column_target_table is not None and \
-                self._add_drop_column_target_table != self._add_drop_column_default_target_table:
-            self._set_current_disruption(f'AddDropColumnMonkey Singular {self.target_node}')
-            self._add_drop_column_run_in_cycle()
-            return
-        self._set_current_disruption(f'AddDropColumnMonkey Self-stress {self.target_node}')
-        if not self._add_drop_column_create_table():
+        if self._add_drop_column_target_table is None:
+            self.log.error("AddDropColumnMonkey: can't find table to run on")
             return
```

Comment:
raise the NemesisNotSupportted event, don't just print to log

Reply:
Fixed